### DISCUSSION TITLE
Localhost not starting

### DISCUSSION THREAD

**Discussion by Sumandeep-Kaur** [sButtons/sbuttons#1471] [General]
When I run command npm start, localhost doesn't start but below text is displayed!

*****
>  sbuttons@1.7.1 start C:\Users\Dell\Desktop\sbuttons
>  npm run watch:less

>  sbuttons@1.7.1 watch:less C:\Users\Dell\Desktop\sbuttons
>  onchange -v src/**/*.less -- npm run compile

>  onchange: watcher ready
*****

Can anyone please help!!

**Comment by shahednasser** [comment]
Hi @Sumandeep-Kaur . In this repository, we have 2 projects:

1. The one in the root refers to the buttons in sbuttons. The scripts in `package.json` are all related to compiling the less files and so on.
2. The one under `website/` is the code for the website. If you want to run the website, you have to change to the directory `website` then `npm install` and run `npm start`
